arm/Microsoft.ApiManagement/service/deploy.bicep

```diff
@@ -425,3 +435,6 @@ output serviceResourceId string = apiManagementService.id
 
 @description('The resource group the API management service was deployed into')
 output serviceResourceGroup string = resourceGroup().name
+
+@description('The principal ID of the system assigned identity.')
+output principalId string = systemAssignedIdentity ? apiManagementService.identity.principalId : ''
```

Comment:
I'm not against this name but I see the storage account uses another `assignedIdentityID`. I'd suggest to use the same. We'll anyway need to align output names at some point (but not for this release)

Reply:
Will align all that use MI to this approach